cmake/FindCLANG_FORMAT.cmake

```diff
@@ -7,6 +7,8 @@
 
 find_program(CLANG_FORMAT_EXECUTABLE
              NAMES clang-format
+	           clang-format-10
+		   clang-format-9
```

Comment:
"Space war" 🤣

Reply:
yeah that is the one problem with clang-format, they are not super compatible

Reply:
Oh I actually meant that these two lines were not aligned with others since they contain tabs.

Reply:
good point